src/transformers/models/detr/feature_extraction_detr.py

```diff
@@ -673,7 +673,7 @@ def pad_and_create_pixel_mask(
 
     # POSTPROCESSING METHODS
     # inspired by https://github.com/facebookresearch/detr/blob/master/models/detr.py#L258
-    def post_process(self, outputs, target_sizes):
+    def post_process(self, outputs, target_sizes, threshold=0.9):
         """
         Converts the output of :class:`~transformers.DetrForObjectDetection` into the format expected by the COCO api.
         Only supports PyTorch.
```

Comment:
This part seems pretty important (COCO api format). Are we sure we're not breaking anything ?

This is what I feared when I mentioned it might be hard to change this.